_posts/2020-07-14-thinking-beyond-the-map.md

```diff
@@ -94,8 +94,8 @@ While traditionally we think of GIS as a tool for making maps and creating and m
 
 #### Links related to AGRC's efforts
 
-- [Partnerships and Sharing]({% link gis-data-sharing/index.md %})
-- [Data sharing Initiatives - Farm to AGOL project]({% link gis-data-sharing/open-data/index.md %})
+- [Partnerships and Sharing]({% link sharing/index.md %})
+- [Data sharing Initiatives - Farm to AGOL project]({% link about/policy/open-data/index.md %})
```

Comment:
**thought**: should the data sharing initiative link go to the policy page?

Reply:
probably not- this was just a find-and-replace to get the website to build after renaming `gis-data-sharing`. Checking the links site-wide will come at the end once the structure is solidified.